src/ui/ui.pro

```diff
@@ -74,6 +74,7 @@ SOURCES += main.cpp\
     EditorNS/bannerbasicmessage.cpp \
     EditorNS/bannerfileremoved.cpp \
     EditorNS/customqwebview.cpp \
+	EditorNS/languagecache.cpp \
```

Comment:
Seems like indentation is wrong here. Tabs vs spaces. Same a little farther below.